src/plugins/orangehrmPerformancePlugin/entity/PerformanceTrackerLog.php

```diff
@@ -42,78 +54,69 @@ class PerformanceTrackerLog
      * @var string|null
      *
      * @ORM\Column(name="log", type="string", length=150, nullable=true)
-     * @ORM\GeneratedValue(strategy="AUTO")
      */
     private ?string $log;
 
     /**
      * @var string|null
      *
      * @ORM\Column(name="comment", type="string", length=3000,nullable=true)
-     * @ORM\GeneratedValue(strategy="AUTO")
      */
     private ?string $comment;
 
     /**
      * @var int|null
      *
      * @ORM\Column(name="status", type="integer", length=11, nullable=true)
-     * @ORM\GeneratedValue(strategy="AUTO")
      */
     private ?int $status;
 
     /**
      * @var string|null
      *
      * @ORM\Column(name="achievement", type="string", length=45,nullable=true)
-     * @ORM\GeneratedValue(strategy="AUTO")
      */
     private ?string $achievement;
 
     /**
      * @var DateTime|null
      *
      * @ORM\Column(name="added_date", type="datetime",nullable=true)
-     * @ORM\GeneratedValue(strategy="AUTO")
      */
-    private ?DateTime $added_date;
+    private ?DateTime $addedDate = null;
 
     /**
      * @var DateTime|null
      *
      * @ORM\Column(name="modified_date", type="datetime",nullable=true)
-     * @ORM\GeneratedValue(strategy="AUTO")
      */
-    private ?DateTime $modified_date;
+    private ?DateTime $modifiedDate = null;
 
 
     /**
      * @var PerformanceTracker|null
      *
-     * @ORM\ManyToOne (targetEntity="OrangeHRM\Entity\PerformanceTracker", inversedBy="PerformanceTrackerLog")
-     * @ORM\JoinColumns({
-     *   @ORM\JoinColumn(name="performance_track_id", referencedColumnName="id")
-     * })
+     * @ORM\ManyToOne (targetEntity="OrangeHRM\Entity\PerformanceTracker")
+     * @ORM\JoinColumn(name="performance_track_id", referencedColumnName="id")
+     *
      */
     private ?PerformanceTracker $performanceTracker;
 
     /**
      * @var Employee|null
```

Comment:
are you sure these fields nullable

Reply:
It is weird but except for id all fields are default NULL

Reply:
shall we make not nullable at entity level
```suggestion
     * @var Employee
```
and change the PHP type `?Employee` ==> `Employee`

same for `$user` field also